docs/hooks/use-text-as-image.md

```diff
@@ -44,7 +44,7 @@ import {useTextAsImage} from 'bandicoot'
 
 function AtomicElement() {
   const [unmodifiableText, setUnmodifiableText] = useState('')
-  const {insertTextAsImage} = useTextAsImage({processSerializedElement})
+  const {insertTextAsImage} = useTextAsImage({processSerializedElement, fontFamily: 'Arial'})
```

Comment:
Maybe leave `fontFamily` off of the example, since for most use cases the default fontFamily calculated by bandicoot is sufficient